src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-tutorial-links.test.ts

```diff
@@ -2,6 +2,16 @@ import nock from 'nock'
 import remark from 'remark'
 import { rewriteTutorialLinksPlugin } from 'lib/remark-plugins/rewrite-tutorial-links'
 
+/**
+ * As we onboard more products into internal beta, we lose the ability
+ * to test "beta" functionality using real config. So, we mock
+ * get-is-beta-product in order to provide a consistent testing config.
+ */
+jest.mock('../../../get-is-beta-product', () => (productSlug) => {
```

Comment:
The `nonBetaProductLinkWithBetaProductInTitle` test case here broken with Boundary now in Beta, and I wasn't sure how to replace, but also didn't want to remove it entirely...

Wasn't exactly sure how to handle this - maybe we should have like `config/test.json` or something?

For now, mocking `get-is-beta-product` seemed like a decent solution.

Reply:
Nice, that works for me! We're probably gonna remove this rewrite plugin during the rollout process (I hope) so this mock makes sense for now.